### PR TITLE
feat: add support for deleting keys

### PR DESCRIPTION
**Pull request by bruuuuuuuce**
This should not get merged until https://github.com/momentohq/cacheservice/pull/654 gets merged and deployed

### CODE REVIEW

**Review by poppoerika — APPROVED** [review]
Looks good to me. 
As you mentioned, we want to wait for merging this until the cacheservice pr to get merged and also the newly added test to pass 👍

**Review by poppoerika — APPROVED** [review]

**Review by tylerburdsall — APPROVED** [review]

**Pull request merged**
